src/Symfony/Component/Mime/Tests/Test/Constraint/EmailAddressContainsTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Tests\Test\Constraint;
+
+use PHPUnit\Framework\ExpectationFailedException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Header\Headers;
+use Symfony\Component\Mime\Test\Constraint\EmailAddressContains;
+
+class EmailAddressContainsTest extends TestCase
+{
+    public function testToString()
+    {
+        $constraint = new EmailAddressContains('headerName', 'expectedValue');
+
+        $this->assertSame('contains address "headerName" with value "expectedValue"', $constraint->toString());
+    }
+
+    public function testFailureDescription()
+    {
+        $headerName = 'headerName';
```

Comment:
I would use something like e-mail address as this is not about an e-mail header

Reply:
Not sure to totally understand what you meant, but I tried an update. Is it better to you?